app/models/reports/repository.rb

```diff
@@ -90,10 +90,16 @@ def warm_cache
     end
 
     def follow_ups_v2_query(group_by: nil)
+      group_field = case group_by
+        when /user_id/ then :user_id
+        when /gender/ then :patient_gender
```

Comment:
This allows for grouping by gender now.

Reply:
I dig this. These regex _could_ use a `^` and `$` but that's nitpicky :)

Reply:
I actually wanted it to be a loose regex, because some clients will group_by `blood_pressures.user_id` and others may ask for `user_id` for example.  I can add the end of string token tho.